test/components/DatasetView.spec.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import { store } from 'store/index';
+import DatasetView from 'components/DatasetView';
+
+const props = {
+  classes: {},
+};
+
+function setup(ownProps = props) {
+  return mount(
+    <Provider store={store}>
+      <DatasetView {...ownProps} />
+    </Provider>,
+  );
+}
+
+describe('DatasetView', () => {
+  const wrapper = setup();
+
+  it('hey look a datasets view', () => {
+    wrapper.find('#datasets').simulate('click');
```

Comment:
I think better to assert that the wrapper can find the element here. If we click something in a test we should verify that the component did something right when it got clicked.